examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -0,0 +1,1815 @@
+{
```

Comment:
<p>"Here we will pretend that we have observed counts of five different species of tree<strong>s</strong> in <s>a</s> <strong>several</strong> forest<strong>s</strong>." (if I understood correctly)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>